packages/web/src/components/CheckboxField/_CheckboxField.scss

```diff
@@ -0,0 +1,90 @@
+@use 'theme';
+@use '../../tools/typography';
+
+.CheckboxField {
+    max-width: 100%;
+}
+
+.CheckboxField__label {
+    @include typography.generate(theme.$label-typography);
+
+    margin-bottom: theme.$spacing;
+    color: theme.$label-color-default;
+}
+
+.CheckboxField__label--hidden {
+    position: absolute;
+    overflow: hidden;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap;
+    border: 0;
+}
```

Comment:
This code is the same as in `TextField` and we will need it in even more cases. Could you please abstract it and reuse in both places? Something like `hide-text()` mixin placed in `src/tools/_accessibility.scss`.

Reply:
Done.